test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py

```diff
@@ -374,3 +367,35 @@ def check_telemetry(ec2_connection, container_name):
     ec2_connection.run(
         f"docker exec -i {container_name} bash -c '[ -f /tmp/test_tag_request.txt ]'"
     )
+
+
+def pull_tensorrt_build_image(ec2_connection, framework_version):
+    """
+    Download tensorflow/tensorflow:<framework_version>-gpu image used for tensorrt model builds.
+    Since tensorflow/tensorflow does not always have all patch versions of a particular TF version,
+    we want to download the closest available version below the patch version of the TF DLC image.
+
+    :param ec2_connection: fabric.Connection object
+    :param framework_version: str framework version of TF image being tested
+    :return: str tensorrt build image tag
+    """
```

Comment:
Added a generic function to deal with unknowns in tensorflow/tensorflow version availability.